security/tor/src/opnsense/service/scripts/tor/make_hidden_service_dirs.php

```diff
@@ -0,0 +1,18 @@
+#!/usr/local/bin/php
+<?php
+
```

Comment:
missing copyright header

Reply:
@fichtner added the header to this and the two other files